fixes.inc

```diff
@@ -3794,6 +3816,22 @@ in the output code.
 
 #define _FIXES_MIN_WORLDBOUNDS (-20000.0)
 
+/**
+ * <remarks>
+ * CARMODTYPE_FRONT_BULLBAR
+ * Defines additional component slots
+ * </remarks>
+ */
+#define CARMODTYPE_FRONT_BULLBAR 14
+
+/**
+ * <remarks>
+ * CARMODTYPE_REAR_BULLBAR
+ * Defines additional component slots
+ * </remarks>
+ */
+#define CARMODTYPE_FRONT_BULLBAR 15
```

Comment:
`REAR`

Reply:
Saw this when I came from gym. I already finished a new version with most of fixed things. But now reading what you wrote